### PR TITLE
[4.0] Media Manager fix for svg files

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/31229

### Summary of Changes
Use the Component options for the images

### Testing Instructions
Follow the tasing instructions on the issue https://github.com/joomla/joomla-cms/issues/31229

### CODE REVIEW

File: administrator/components/com_media/resources/scripts/components/browser/items/item.es6.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import File from './file.vue';
 import Image from './image.vue';
 import Video from './video.vue';
 import * as types from '../../../store/mutation-types.es6';
+import { api }  from '../../../app/Api.es6'
```
Comment (SharkyKZ) [inline administrator/components/com_media/resources/scripts/components/browser/items/item.es6.js]:
```suggestion
import { api } from '../../../app/Api.es6';
```

**Comment by Quy** [comment]
How to fix with the untranslated string when not one of the image extensions such as PDF?

**Comment by dgrammatiko** [comment]
> How to fix with the untranslated string when not one of the image extensions such as PDF?

Well what I did here needs to be repeated for all the types, eg video, audio, documents

```js
const imageExtensions = api.imagesExtensions;
const videoExtensions = api.videoExtensions;
const audioExtensions = api. audioExtensions;
const docExtensions = api. docExtensions;
```

Of course this requires the appropriate changes in the Api.js and some fields in the com_media options to handle video, audio and docs...
And then some more changes in the vue files to handle properly all these files (eg the fix in this PR is not fixing the missing thumb for the svg)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5eec638a033286a18c7135e641bad7109d51c293<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31233">issues.joomla.org/tracker/joomla-cms/31233</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 5eec638a033286a18c7135e641bad7109d51c293<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31233">issues.joomla.org/tracker/joomla-cms/31233</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31233">issues.joomla.org/tracker/joomla-cms/31233</a>.</sub>

**Comment by infograf768** [comment]
@dgrammatiko 
api question for ignorant people:
Where are the `options.imagesExtensions` (and similar for video which is for now limited to mp4, etc.) fetched from in Api.es6 ? Link to the code please.

**Comment by dgrammatiko** [comment]
@infograf768 follow the files touched here, eg
Api: line 26 in administrator/components/com_media/resources/scripts/app/Api.es6.js
which is the data that is passed in the template: line 47 in administrator/components/com_media/tmpl/media/default.php
which is the options from https://github.com/joomla/joomla-cms/blob/5eec638a033286a18c7135e641bad7109d51c293/administrator/components/com_media/config.xml#L68-L76

Obviously video, audio and docs doesn't exist in the config.xml neither in the template neither in the Api (in the Api everything was hardcoded before this PR, but still there is work to be done for video, audio and docs)

**Comment by infograf768** [comment]
Got it. Thanks.

Merging this and hopefully you get some time to follow up. ;)

**Comment by infograf768** [comment]
Tks.

**Pull request merged**